tripy/tests/integration/test_groupnorm.py

```diff
@@ -42,18 +44,58 @@ def test_groupnorm_accuracy(self, torch_dtype, tp_dtype, input_shape, num_groups
             num_groups=num_groups,
             num_channels=num_channels,
             eps=eps,
-            dtype=tp_dtype,
+            dtype=dtype,
         )
 
+        input = torch.empty(*input_shape, dtype=torch_dtype, device="cuda").uniform_(0, 10)
+        tp_input = tp.Tensor(input, dtype=dtype)
+
+        # Verify normalized output has approximately mean=0, std=1
+        tp_groupnorm.weight = tp.ones((num_channels,), dtype=dtype)
+        tp_groupnorm.bias = tp.zeros((num_channels,), dtype=dtype)
+
+        output = eager_or_compiled(tp_groupnorm, tp_input)
+        output_torch = torch.from_dlpack(output)
+
+        N, C = input_shape[0], input_shape[1]
+        spatial_size = torch.prod(torch.tensor(input_shape[2:]))
+        reshaped = output_torch.view(N, num_groups, C // num_groups, spatial_size)
+
+        means = reshaped.mean(dim=(2, 3))
+        vars = reshaped.var(dim=(2, 3), unbiased=False)
+
+        mean_abs = means.abs().mean().item()
+        var_diff = (vars - 1).abs().mean().item()
+
+        assert mean_abs < 2e-4, f"Group mean should be close to 0, got {mean_abs}"
+        assert var_diff < 1e-3, f"Group variance should be close to 1, got {var_diff}"
+
+        # Comparison test with the affine transformation included
```

Comment:
Seems like this should be a separate test. We can use a fixture to reuse the modules/inputs.